.env.example

```
@@ -1 +1,73 @@
+TRADE_SERVER_PORT=
+ASSET_SERVER_PORT=
+STOCK_SERVER_PORT=
+PORTFOLIO_SERVER_PORT=
+COMPETITION_SERVER_PORT=
+RANKING_SERVER_PORT=
+AI_SERVER_PORT=
+NOTIFICATION_SERVER_PORT=
+
+# db schema
+USER_SCHEMA=
+TRADE_SCHEMA=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TRADE_SCHEMA key should go before the USER_SCHEMA key

(UnorderedKey)